src/main/java/pro/sky/collections/listssets/service/DepartmentServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package pro.sky.collections.listssets.service;
+
+import org.springframework.stereotype.Service;
+import pro.sky.collections.listssets.model.Employee;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+@Service
+public class DepartmentServiceImpl implements DepartmentService{
+    private final EmployeeServiceImpl employeeService;
```

Comment:
```suggestion
    private final EmployeeService employeeService;
```
инъекцию нужно делать всегда по интерфейсу